docs/Reference.md

```diff
@@ -363,6 +363,28 @@ data:image/jpeg;base64,BASE64_ENCODED_JPEG_IMAGE_DATA
 
 Ensure you use the proper content type (`image/jpeg`, `image/png`, `image/gif`) that matches the image data being provided.
 
+### Signed Attachment CDN URLs
+
+Attachments uploaded to Discord's CDN (like user and bot-uploaded images) have signed URLs with a preset expiry time. Discord automatically refreshes attachment CDN URLs that appear within the client, so when your app receives a payload with a signed URL (like when you [fetch a message](#DOCS_RESOURCES_CHANNEL/get-channel-message)), it will be valid.
```

Comment:
Are messages with attachment links in the content updated, at least for the client?